test/image-tests/aws.sh

```diff
@@ -0,0 +1,256 @@
+#!/bin/bash
+set -euo pipefail
+
+source /etc/os-release
```

Comment:
I like this very much! I think we should make it more consistent with the other tests, shipping it as just another test binary, but all that we can discuss in the future, let's merge it as is.

Reply:
Agreed. I tried my best to make it a test that someone could just run from another machine without needing Jenkins.